test/helpers/position_assertions.cc

```diff
@@ -763,6 +769,39 @@ string TypeAssertion::toString() const {
     return fmt::format("type: {}", symbol);
 }
 
+FileDefAssertion::FileDefAssertion(std::string_view filename, std::unique_ptr<Range> &range, int assertionLine,
+                                   std::string targetFilename, int targetLine, int targetColumn)
+    : BaseDefAssertion(filename, range, assertionLine, ""sv), targetFilename(targetFilename), targetLine(targetLine),
+      targetColumn(targetColumn) {}
+
+std::shared_ptr<FileDefAssertion> FileDefAssertion::make(std::string_view filename, std::unique_ptr<Range> &range,
+                                                         int assertionLine, std::string_view assertionContents,
+                                                         std::string_view assertionType) {
+    vector<string_view> split = absl::StrSplit(assertionContents, ' ');
+    CHECK_EQ(split.size(), 3);
+    {
+        INFO(fmt::format("Invalid file-def assertion; multiple words found:\n{}\nFile definition assertions should be "
```

Comment:
`INFO` has to go before `CHECK_EQ` and both have to be in the same scope. `INFO` is printed if any checks within the scope fail.